### PR TITLE
F21-612 - Fix: long farm name overflow on the farm selection page 

### PR DESCRIPTION
**Pull request by anant00729**
To Test: 

1. Go to http://localhost:3000/farm_selection page.
2. Inspect the page in your browser:-
3. Add some dummy text as shown below. 

Screen Shot 2022-06-01 at 12 17 25 PM

Screen Shot 2022-06-01 at 12 39 09 PM

For details: https://lite-farm.atlassian.net/browse/F21-612

Please check the CSS compatibility of word-break: https://caniuse.com/?search=word-break

### CODE REVIEW

**Review by seinnlee — APPROVED** [review]
It looks good to me

**Review by smattingly — CHANGES_REQUESTED** [review]
I had one question on this. No change needed if you judge this is best-- just resubmit and I will merge.

File: packages/webapp/src/components/ChooseFarm/ChooseFarmMenu/ChooseFarmMenuItem/index.jsx [hunk 1]
```diff
@@ -25,12 +25,16 @@ const ChooseFarmMenuItem = ({
...
+        wordBreak: 'break-word',
+        padding: '8px 0',
         ...style,
       }}
       {...props}
     >
       <div className={clsx(styles.leftColumn, styles[color])}>
-        <h5 className={clsx(styles.farmName)}>{farmName}</h5>
+        <h5 className={clsx(styles.farmName)}>
+          {farmName.length > 77 ? `${farmName.substring(0, 77).trim()}...` : farmName}
```
[the remark this answers is not in the document]
Comment (anant00729) [reply packages/webapp/src/components/ChooseFarm/ChooseFarmMenu/ChooseFarmMenuItem/index.jsx:36]: I thought of using the following css properties: 

display: -webkit-box;
-webkit-line-clamp: 3;
-webkit-box-orient: vertical;
overflow: hidden;
text-overflow: ellipsis;

this restricts the content to 3 lines. but it doesn't guarantee ellipsis after 77 characters. The content can vary from 2 lines to 3 lines depending on the responsiveness. 

Please let me know if you would like me to use CSS or handle in JS  

@smattingly

[the remark this answers is not in the document]
Comment (anant00729) [reply packages/webapp/src/components/ChooseFarm/ChooseFarmMenu/ChooseFarmMenuItem/index.jsx:36]: Also please have a look at the CSS compatibility on various browsers:

https://caniuse.com/?search=-webkit-line-clamp%20

[the remark this answers is not in the document]
Comment (smattingly) [reply packages/webapp/src/components/ChooseFarm/ChooseFarmMenu/ChooseFarmMenuItem/index.jsx:36]: I'll merge it as written. I just wanted to ask if the ellipsis was better.

**Pull request merged**